code/game/machinery/computer/camera.dm

```diff
@@ -233,6 +232,16 @@
 	else
 		set_light(1, LIGHTING_MINIMUM_POWER)
 
+/obj/machinery/computer/security/telescreen/entertainment/wrench_act(mob/living/user, obj/item/I)
+	. = TRUE
+	if(!I.use_tool(src, user, 0 SECONDS))
+		return
+	TOOL_ATTEMPT_DISMANTLE_MESSAGE
+	if(I.use_tool(src, user, 2 SECONDS, volume = I.tool_volume))
+		TOOL_DISMANTLE_SUCCESS_MESSAGE
+		new /obj/item/mounted/frame/display/entertainment_frame(drop_location(), 1)
```

Comment:
```suggestion
		new /obj/item/mounted/frame/display/entertainment_frame(drop_location())
```
This `1` ain't a parameter used here as far as I can see.
Is this a left over from `new /obj/item/stack/sheet/metal(drop_location(), 1)`? Since here it means to make 1 metal stack with 1 metal in it

Reply:
it is leftover, this typepath only takes 1 argument (loc)